extension-compose/src/main/java/com/mapbox/maps/extension/compose/annotation/generated/PointAnnotationGroup.kt

```diff
@@ -90,6 +92,7 @@ public fun PointAnnotationGroup(
   iconTranslateAnchor: IconTranslateAnchor? = null,
   textTranslate: List<Double>? = null,
   textTranslateAnchor: TextTranslateAnchor? = null,
+  onAnnotationManagerCreated: (PointAnnotationManager) -> Unit = {},
```

Comment:
Hey @hiroaki404 , thanks for contributing. With the proposed approach, I think you still cannot get the annotation id, as the annotation is supposed to be created by compose and you should not directly call `PointAnnotationManager.create()`. 
I think we'd go with exposing the feature id setter in the AnnotationOptions class, as it's more friendly in compose context and also align with iOS implementation. I have created a internal ticket for this feature.

Reply:
> I think we'd go with exposing the feature id setter in the AnnotationOptions class, as it's more friendly in compose context and also align with iOS implementation. I have created a internal ticket for this feature.

If that's possible, I think it's better too. Thank you!